monkey/infection_monkey/exploit/shellshock.py

```diff
@@ -182,6 +187,31 @@ def attempt_exploit(self, url, attacks=_attacks):
                 LOG.debug("URL %s does not seem to be vulnerable with %s header" % (url, header))
         return False,
 
+    @classmethod
+    def _try_lock(cls, exploit, url, header):
+        """
+        Checks if another monkey is running shellshock exploit
+        :return: True if no monkey is running shellshock exploit
+        """
+        file_path = '/tmp/monkey_lock'
```

Comment:
Move '/tmp/monkey_lock' to the top of the file, next to other const. Use it here and in _exit_lock method.